Makefile

```diff
@@ -47,7 +47,8 @@ venv/%:
 #=> develop: install package in develop mode
 .PHONY: develop setup
 develop setup:
-	pip install -e .[dev,extras,notebooks]
+	pip install -e .[dev,extras,notebooks]; \
+	pre-commit install
```

Comment:
this seems to work for (and make sense to) me but I'm always wary of makefile environment construction soooo